tests/MaintainersTest.php

```diff
@@ -258,86 +258,6 @@ public function testSettingAutoJoinFlag()
         $this->assertEquals(false, $org['allowAutoJoin']);
     }
 
-    /**
-     * @dataProvider autoJoinProvider
-     * @param bool $allowAutoJoin
-     */
-    public function testMaintainerAutoJoinError(bool $allowAutoJoin)
-    {
-        $organization = $this->client->createOrganization($this->testMaintainerId, [
-            'name' => 'Test org',
-        ]);
-        $this->client->updateOrganization($organization['id'], [
-            'allowAutoJoin' => $allowAutoJoin,
-        ]);
-
-        // make sure normalUser is a maintainer
-        $this->addNormalMaintainer();
-
-        $testProject = $this->client->createProject($organization['id'], [
-            'name' => 'Test Project',
-        ]);
-
-        $projectUser = $this->findProjectUser($testProject['id'], $this->normalUser['email']);
-        $this->assertNull($projectUser);
-
-        try {
-            $this->normalUserClient->addUserToProject($testProject['id'],[
-                "email" => $this->normalUser['email']
-            ]);
-            $this->fail('Project join should produce error');
-        } catch (ClientException $e) {
-            $this->assertEquals(403, $e->getCode());
-        }
-
-        $projectUser = $this->findProjectUser($testProject['id'], $this->normalUser['email']);
-        $this->assertNull($projectUser);
-    }
-
-    /**
-     * @dataProvider autoJoinProvider
-     * @param bool $allowAutoJoin
-     */
-    public function testSuperAdminAutoJoinError(bool $allowAutoJoin)
```

Comment:
Nahrazeno `ProjectJoinTest::testSuperAdminCannotAddSelfToProjectRegardlessOfAllowAutoJoin()`

Tady to imho nemelo byt.